tests/tests_enabled.yml

```diff
@@ -2,11 +2,11 @@
 - name: Ensure that the role runs with rsyslog_enabled=True
   hosts: all
   become: true
+  vars:
+    rsyslog_default: True
```

Comment:
lowercase

Reply:
the change is still there.

Reply:
(;_;) Thanks for finding it out, @pcahyna.  It seems I might have made some mistakes in the conflict resolution...  I'm planning to open another bit PR for changing the internal variables to start with '_'.  Is it ok to add the fix to the PR?

Reply:
@nhosoi I suggest that you just force-push to master with this part corrected, I think you will not create problems for anyone by force pushing so soon after you merged the PR @richm what do you think?